mscore/scoreview.cpp

```diff
@@ -714,6 +714,7 @@ void ScoreView::moveControlCursor(const Fraction& tick)
         }
         _timeElapsed += addition;
     } else { // reposition the cursor when the score is not playing
+        std::cout << "here" << std::endl;
```

Comment:
@SKefalidis Could you please delete this?

Reply:
Done.